integration/chartconfig/chartconfig.go

```diff
@@ -0,0 +1,43 @@
+// +build k8srequired
+
+package chartconfig
+
+import (
+	"bytes"
+	"html/template"
+
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+)
+
+type ChartConfigValues struct {
+	Channel              string
+	ConfigMap            ConfigMap
+	Name                 string
+	Namespace            string
+	Release              string
+	Secret               Secret
+	VersionBundleVersion string
+}
+
+type ConfigMap struct {
+	Name            string
+	Namespace       string
+	ResourceVersion string
+}
+
+type Secret struct {
+	Name            string
+	Namespace       string
+	ResourceVersion string
+}
+
+func (ccv ChartConfigValues) ExecuteChartValuesTemplate() (string, error) {
```

Comment:
Again me and data structs. I really don't like having methods like that on pure data structs. I think this one here should be a standalone func. Otherwise we end up with adding things like logger to the data structs.

Reply:
I moved the data structure to the e2etemplates repo. PTAL.